soc/realtek/ec/rts5912/soc.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ * Author: Lin Yu-Cheng <lin_yu_cheng@realtek.com>
+ */
+
+#ifndef _SOC_H_
+#define _SOC_H_
```

Comment:
```suggestion
#ifndef SOC_REALTEK_RTS5912_H_
#define SOC_REALTEK_RTS5912_H_
```

Reply:
We changed the identifiers in `soc.h`, thanks.